### ISSUE TITLE
Benzoxazoles in the MMV boxes

### ISSUE THREAD

**Issue by KlementineJBS** [OpenSourceMycetoma/series-5-benzimidazoles#6]
After discussing at the [October 2024](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/27) meeting, I've been looking through the MMV boxes for benzoxazole structures so that @AndreaOtago and others can consider those when looking at SAR for this series. 

I wasn't able to find any in the Covid Response box, but have checked the GHP, Pathogen and Pandemic Response boxes. I couldn't find a list of structures for the Stasis box (other than as SMILES) so haven't checked that yet.

Data is taken directly from the MMV tab on the masterlist.

![image](https://github.com/user-attachments/assets/0653acb8-46f1-47a7-8737-79a853a74a1e)

[Benzoxazoles in MMV Boxes.xlsx](https://github.com/user-attachments/files/17542029/Benzoxazoles.in.MMV.Boxes.xlsx)

**Comment by AndreaOtago** [comment]
@KlementineJBS thanks for that. The only similar structure is Zoxazolamine, which is worth mentioning in the paper introduction @MA-Jjingyi , I am unsure how we reference the MMV box/results/data from above post. 

Also @KlementineJBS I am not sure how to comment directly on meeting comments/to dos - but I have already written the SAR draft for the paper with @MA-Jjingyi so the comment about can anyone help with this - all under control. The latest MS drafts are posted under a seperate page set up by @MA-Jjingyi

**Comment by KlementineJBS** [comment]
Hey @AndreaOtago - you can either add a comment to the meeting page (for longer responses, images etc) or simply edit the comment with the minutes and add a note. 

Usually I do something like:

- [x] @KlementineJBS to update something <-- done, link is here

If you're not able to edit the comment, tag Mat and he should be able to check if there's an issue with permissions. But leaving your own comment is always fine. 

RE: SAR --Just so we're all on the same page (if you'll forgive the pun), is [this](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/issues/5) the page you're referring to? I don't think this page was shared with the larger group so we hadn't seen any updates for a while and were just wondering why!

**Comment by AndreaOtago** [comment]
> Hey @AndreaOtago - you can either add a comment to the meeting page (for longer responses, images etc) or simply edit the comment with the minutes and add a note.
> 
> Usually I do something like:
> 
> * [x]  @KlementineJBS to update something <-- done, link is here
> 
> If you're not able to edit the comment, tag Mat and he should be able to check if there's an issue with permissions. But leaving your own comment is always fine.
> 
> RE: SAR --Just so we're all on the same page (if you'll forgive the pun), is [this](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/issues/5) the page you're referring to? I don't think this page was shared with the larger group so we hadn't seen any updates for a while and were just wondering why!

@KlementineJBS yes that is the link that we have a couple of drafts posted into.
